app/src/Containers/Home/index.tsx

```diff
@@ -1,19 +1,25 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { BeautifulText, SexyContainer } from "./styles";
+import Header from "../Header";
+import JokeForm from "../JokeForm";
+import qs from "qs"
 
-const req = async () => {
+const reqJoke = async () => {
```

Comment:
Another thought was using actions and reducers to make these api calls and setting a global value to the "Joke of the day" In case it was to be used somewhere besides the homepage.